foundation/foundation-upgrade/src/main/java/com/boozallen/aissemble/upgrade/migration/v1_8_0/AiopsReferencePythonMigration.java

```diff
@@ -0,0 +1,88 @@
+package com.boozallen.aissemble.upgrade.migration.v1_8_0;
+
+/*-
+ * #%L
+ * aiSSEMBLE::Foundation::Upgrade
+ * %%
+ * Copyright (C) 2021 Booz Allen
+ * %%
+ * This software package is licensed under the Booz Allen Public License. All Rights Reserved.
+ * #L%
+ */
+
+import com.boozallen.aissemble.upgrade.migration.AbstractAissembleMigration;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.technologybrewery.baton.BatonException;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.util.*;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import static com.boozallen.aissemble.upgrade.util.FileUtils.replaceLiteralInFile;
+
```

Comment:
S: we'd like all classes to have a class-level comment.  I generally used anything public or not obvious as my markers to add javadoc